core/domain/exp_jobs_one_off_test.py

```diff
@@ -957,7 +958,7 @@ def test_for_migration_job(self):
         state2.update_interaction_default_outcome(default_outcome_dict2)
         exp_services.save_new_exploration(self.albert_id, exploration)
 
-        # Start validation job on sample exploration.
+        # Start migrationvalidation job on sample exploration.
```

Comment:
nit: `MigrationValidation` for readability

Reply:
Right, done!